portal/tests/pageObjects/portal/teach/onboarding_organisation_page.py

```diff
@@ -96,6 +96,11 @@ def has_creation_failed(self):
         error = 'There is already a school or club registered with that name and postcode'
         return error in errors
 
+    def was_postcode_invalid(self):
+        errors = self.browser.find_element_by_id('form-create-organisation').find_element_by_class_name('errorlist').text
+        error = 'Please enter a valid postcode or ZIP code'
```

Comment:
we should generate our own exception classes and make sure we test that instead if copy pasting messages like that :) - improvement for later I guess. @faucomte @OlafSzmidt

Reply:
@OlafSzmidt can you create an issue about that and we can merge. Thanks :)

Reply:
#646 opened. @CelineBoudier